resources/quality/ultimaker_s3/um_s3_aa0.4_abs_0.3mm.inst.cfg

```
@@ -0,0 +1,24 @@
+[general]
+definition = ultimaker_s3
+name = Extra Fast - Experimental
+version = 4
+
+[metadata]
+is_experimental = True
+material = generic_abs
+quality_type = verydraft
+setting_version = 22
+type = quality
+variant = AA 0.4
+weight = -3
+
+[values]
```

Need to enable the acceleration since it is now disabled by default
```suggestion
[values]
_plugin__curaenginegradualflow__0_1_0__gradual_flow_enabled = True
```